.eslintignore

```diff
@@ -2,3 +2,4 @@ node_modules
 .nyc_output
 coverage
 dist
+test/fixtures/syntax-error-module.js
```

Comment:
Ah, nevermind, I expect it will probably blow up before even parsing that comment right?